app/views/components/Product/ProductItem.tsx

```diff
@@ -0,0 +1,72 @@
+import { StyleSheet, Text, View, TouchableHighlight } from "react-native";
+import React from "react";
+import { useNavigation } from "@react-navigation/native";
+
+import { colors } from "../../styles";
+
+function ProductItem(props) {
+  const { productDetails } = props;
+  const navigation = useNavigation();
+
+  return (
+    <TouchableHighlight
+      style={styles.item}
+      //   onPress={() => {
+      //     navigation.navigate("ProductDetails", { productDetails });
+      //   }}
+      underlayColor={colors.sb_orange_100}
+    >
+      <>
+        <View style={styles.image}></View>
+        <View>
+          <Text style={styles.name}>{productDetails.name}</Text>
+          <View style={styles.detailRow}>
+            <Text style={styles.quantity}>500g</Text>
+            <Text style={styles.price}>Rs. {productDetails.price}/-</Text>
+          </View>
+        </View>
+      </>
+    </TouchableHighlight>
+  );
+}
+
+const styles = StyleSheet.create({
+  item: {
+    flex: 1,
+    margin: 3,
+    height: 80,
+    borderRadius: 5,
+    backgroundColor: colors.sb_orange_200,
+    flexDirection: "row",
+    padding: 10,
+  },
+  name: {
+    fontSize: 20,
+    fontWeight: "500",
+    color: colors.sb_dark,
+    marginTop: 8,
+  },
+  image: {
+    height: 60,
+    width: 60,
+    backgroundColor: colors.sb_gray_100,
+    marginRight: 10,
+    borderRadius: 5,
+  },
+  detailRow: {
+    flex: 1,
+    flexDirection: "row",
+  },
+  quantity: {
+    fontSize: 17,
+    marginTop: 7,
+  },
+  price: {
+    fontSize: 20,
+    marginTop: 5,
+    marginLeft: 10,
+    fontWeight: "600",
+  },
```

Comment:
Let's set it as right alignment and bigger fonts (as shown in Hifis)
Because user's are interested mainly on product with amount so let's highlight it. 

![image](https://github.com/ndedhia1210/KiranaShop/assets/127356489/9a19b4e7-79af-4dc9-9861-c89d68193748)

Reply:
Done. Updated PR. Here's how it looks now.
![IMG_8585](https://github.com/ndedhia1210/KiranaShop/assets/14354751/dcf6bf13-3061-4a12-b24f-b9261629c8bb)